www/include/common/javascript/datepicker/localizedDatepicker.js

```diff
@@ -62,19 +65,35 @@ function initDatepicker(className, altFormat, defaultDate, idName, timestampToSe
             var altName = jQuery(this).attr("name");
             if (typeof(altName) != "undefined") {
                 var alternativeField = "input[name=alternativeDate" + altName[0].toUpperCase() + altName.slice(1) + "]";
-                var value = $(this) && $(this).val() ? $(this).val() : defaultDate;
+                var value;
+                // avoid to loose chosen localized values on refresh
+                if ($(alternativeField) && $(alternativeField).val()) {
+                    // alternativeField value has a MM/DD/YYYY format (the engine supported format)
+                    value = moment($(alternativeField).val(), "MM/DD//YYYY");
+                } else if ($(this) && $(this).val()) {
+                    // $(this).val(), if exists, is a GMT YYYY-MM-DDTHH:mm:ss timestamp
+                    // for example with PHP : gmdate("Y-m-d\TH:i:s")
+                    value = moment($(this).val()).tz(timezone);
+                } else {
+                    value = defaultDate;
+                }
```

Comment:
Why not define directly:

 var value = defaultDate;

and then do the if / else if

Reply:
indeed this make sense. I'll add it in another PR to avoid reloading the validation and CI processes -> https://github.com/centreon/centreon/pull/8490